core/blockstm/executor_test.go

```diff
@@ -294,6 +306,66 @@ func testExecutorComb(t *testing.T, totalTxs []int, numReads []int, numWrites []
 	fmt.Printf("Total exec duration: %v, total serial duration: %v, time reduced: %v, time reduced percent: %.2f%%\n", totalExecDuration, totalSerialDuration, totalSerialDuration-totalExecDuration, float64(totalSerialDuration-totalExecDuration)/float64(totalSerialDuration)*100)
 }
 
+// // nolint: gocognit
```

Comment:
Let's try not to leave large chunk of commented code. Are we supposed to uncomment these lines?

Reply:
Is there a way to make them run faster, e.g. reduce the number Txs?

Reply:
Yes, that will work. Also, if it doesn't affect other tests, we can increase the go-test timeout in the CI.

Reply:
Reduced the number of txns.